DATA/MISSIONS/mbases.ini

```diff
@@ -37343,7 +37343,7 @@ rumor = base_0_rank, mission_end, 3, 461615
 ; My assignment takes me to the system with the megastructure on a regular basis. First we establish a path through the opening in front of the gate, then I take half my squadron through. We stay until we have the desired amount of data or until the Nomad response gets too hot. My old tutor at West Point said that there is science in every combat deployment, back then I didn't consider taking that literally.
 rumor = base_0_rank, mission_end, 3, 461616
 ;res html order_toledo_rumor21
-; The inside of the Dyson sphere? <i>Stunning. A testament to the progress and enlightenment which humanity has not yet reached.</i> — That's how Orillion described it to me when I asked him about it. He's one of the few people who have ever been in there. All operations of that kind were in the early days of The Order, when the battalion in this system was only a very small group.
+; The inside of the Dyson sphere? \iStunning. A testament to the progress and enlightenment which humanity has not yet reached.\I — That's how Orillion described it to me when I asked him about it. He's one of the few people who have ever been in there. All operations of that kind were in the early days of The Order, when the battalion in this system was only a very small group.

```

Comment:
Stylistic choice. The bar patron only retroactively justifies this as something he's quoting from Orillion, which made me choose italicised text as a signifier rather than quotation marks or a combination of both. 

If the rumor was saying 'Orillion said: "[...]"', I would have always used quotation marks, but for all we know what the guy is telling you might not even be a direct quote, or at least paraphrased.